wandb/sdk/wandb_run.py

```diff
@@ -559,7 +565,20 @@ def _set_teardown_hooks(self, hooks):
 
     def _set_run_obj(self, run_obj):
         self._run_obj = run_obj
-        # TODO: Update run summary when resuming?
+        # Grab the config from resuming
+        if run_obj.HasField("config"):
+            c_dict = config_util.dict_no_value_from_proto_list(run_obj.config.update)
+            # TODO: Windows throws a wild error when this is set...
+            if "_wandb" in c_dict:
+                del c_dict["_wandb"]
+            # We update the config object here without triggering the callback
+            self.config._update(c_dict, allow_val_change=True)
+        # Update the summary, this will trigger an un-needed graphql request :(
```

Comment:
I'm seeing a good argument for an optional `skip_update` arg in Summary::_update :)